ansible/roles/host-ocp4-hcp-cnv-install/tasks/print_cluster_info.yml

```diff
@@ -0,0 +1,64 @@
+---
+- name: Get console route
+  environment:
+    KUBECONFIG: /home/{{ ansible_user }}/.kube/config
+  ansible.builtin.command: oc get route -n openshift-console console -o json
```

Comment:
This should use some k8s_info command. Either from the console operator or just query the route object.